exonum/src/blockchain/schema.rs

```diff
@@ -335,46 +335,46 @@ where
 }
 
 // TODO Write more meaningful description [ECR-3824]
-/// Describes the membership for the index.
+/// Describes the origin of the schema.
 #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub enum IndexOwner {
-    /// This index is a part of the core schema.
+pub enum SchemaOrigin {
+    /// This is a Core schema.
     Core,
-    /// This index is a part of the runtime schema with the specified ID.
+    /// This is a runtime schema with the specified ID.
     Runtime(u32),
-    /// This index is a part of some service schema with the specified ID.
+    /// This is a service schema with the specified instance ID.
     Service(InstanceId),
 }
 
-impl IndexOwner {
-    /// Creates index coordinate for the current owner.
+impl SchemaOrigin {
+    /// Computes the index coordinates inside the schema.
     pub fn coordinate_for(self, index_id: u16) -> IndexCoordinates {
         IndexCoordinates::new(self, index_id)
     }
 
-    /// Returns the corresponding tag.
-    fn tag(self) -> IndexTag {
+    /// Returns the corresponding origin binary value.
```

Comment:
Maybe "the origin label" instead of "binary value"? Usually "binary value" is associated with byte juggling, which is not exactly the case here.